multy_core/src/golos/golos_transaction.cpp

```diff
@@ -195,8 +264,30 @@ class GolosTransactionTransferOperation : public GolosTransactionOperation
         return TRANSFER;
     }
 
-    void write_to_stream(GolosBinaryStream* /*stream*/) const override
+    void write_to_stream(GolosBinaryStream* stream) const override
     {
+        *stream << (uint8_t)0x02;  // transfer_operation id
+        *stream << static_cast<uint8_t>(from.size());
+        *stream << from;
+        *stream << static_cast<uint8_t>(to.size());
+        *stream << to;
+
+        *stream << amount.get_value_as_uint64();
+        *stream << static_cast<uint8_t>(GOLOS_VALUE_DECIMAL_PLACES);
+
+        std::string temp (7, '\0');
```

Comment:
This looks silly, could you please explain what `7` stands for? also, this could be done much easier:
```
std::string normalized_token_name(token_name);
normalized_token_name.resize(7);
```